test/node/tools/utils.js

```diff
@@ -128,6 +128,14 @@ const bufferFromHexArray = array => {
 
 exports.bufferFromHexArray = bufferFromHexArray;
 
+function int32ToHex(int32) {
```

Comment:
the naming here is a bit misleading, especially without TS.  Since we have a dedicated `Int32` class, I expected this method to take an instance of Int32.  Can we clarify somehow that this is a number? 

One option:

```typescript
function numberToHex(size: 'i32' | 'i64', n) { 
...
}
```

Reply:
I added documentation here, the method can accept an Int32 already b/c of type coercion